lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,12 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
+    def update_destination(destination_branch, remote = 'origin')
       origin_branch = current_branch
       run_command_with_label "git checkout #{destination_branch}"
-      run_command_with_label "git pull origin #{destination_branch}"
+      run_command_with_label "git pull #{remote} #{destination_branch}"
       run_command_with_label "git checkout #{origin_branch}"
+      run_command_with_label "git rebase #{destination_branch}"
```

Comment:
I think it makes sense to keep the rebase/merge outside the context of this method. You don't have to keep everything in my example, but was more of an outline for what I'd expect to happen in a "refresh".  It'd be fine if all this did was `update_destination` and `pull origin #{base_branch}`.  We will need tests for the new method though.